.npmignore

```diff
@@ -17,3 +17,4 @@ coverage/
 coverage/*
 scratch
 scratch/*
+lib
```

Comment:
Please remove this.

Reply:
Please review https://github.com/winstonjs/winston/pull/1418#discussion_r211768849. Removing this will publish two different versions of the source code. I'll happily change it, just wanted to make sure that's what you want :)

Reply:
Yes I would like to include both versions of source.